### DISCUSSION TITLE
BLS signature  in .net library

### DISCUSSION THREAD

**Discussion by SalimiHabib** [dkackman/chia-dotnet#3] [Q&A] [answered]
Hi
chia use `blspy `in python world to access BLS signature, `G2Element`, `G1Element `,and `AugSchemeMPL `verifier method .
Is there any equal option in .NET core ?

thank you

**Comment by dkackman** [comment, marked as answer]
Not directly but this looks like it might be promising for you.
https://github.com/sgryphon/cortex-cryptography-bls
